unit/CMakeLists.txt

```diff
@@ -17,7 +17,11 @@ FetchContent_MakeAvailable(googletest)
 
 enable_testing()
 
-add_executable(unit_test conversion_test.cpp data_test.cpp msg_test.cpp)
+if(ASTARTE_TRANSPORT_GRPC)
+    add_executable(unit_test conversion_test.cpp data_test.cpp msg_test.cpp)
```

Comment:
Some of those tests might still be useful for mqtt. For example, the `msg_test` will probably be valid for both.